src/vmspawn/vmspawn-scope.c

```diff
@@ -60,6 +60,8 @@ int start_transient_scope(sd_bus *bus, const char *machine_name, bool allow_pidf
                                   "Description", "s",  description,
```

Comment:
Hmm, `(sa)` should be replaced by `(sv)` above too...

Reply:
ah, I missed that...